hil-test/tests/lcd_cam_i8080.rs

```diff
@@ -76,10 +77,7 @@ mod tests {
 
     #[test]
     fn test_i8080_8bit_async_channel(ctx: Context<'static>) {
-        let channel = ctx
-            .dma
-            .channel0
-            .configure_for_async(false, DmaPriority::Priority0);
+        let channel = ctx.dma.channel0.configure(false, DmaPriority::Priority0);
```

Comment:
No `into_async()`? Without it this test might as well be removed.

Reply:
is this just an "it compiles" test or does it have other value?

Reply:
The test was checking that using a blocking I8080 with an async DMA channel works as expected. (More info in the i8080 async pr)